src/common/CqlEditorTerminologySection.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
+import "./CqlEditorTerminologySection.scss";
+
+interface CqlEditorTerminologySectionProps {
+  title: string;
+  children?: any;
+}
+
+const CqlEditorTerminologySection = (
+  props: CqlEditorTerminologySectionProps
+) => {
+  const { title, children } = props;
+  const [open, setOpen] = useState(true);
+  const chevronClass = open ? "chevron-display open" : "chevron-display";
+  const growingDivClass = open ? "growing-div open" : "growing-div";
+
+  return (
+    <div
+      className="cql-editor-tab-heading"
+      data-testid={`cql-editor-${props.title}-sub-heading`}
+    >
+      <div
+        onClick={() => {
+          setOpen(!open);
+        }}
+        tabIndex={0}
+        role="button"
+        onKeyPress={(e) => {
```

Comment:
onclick does not cover this?

Reply:
this is something for 508 compliance, onClick is not working when we click enter on the tab heading so onKeyPress is used